fe/fe-core/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -122,7 +122,7 @@ public class FunctionCallExpr extends Expr {
                 Preconditions.checkArgument(children.get(1) instanceof IntLiteral
                         || (children.get(1) instanceof CastExpr
                                 && children.get(1).getChild(0) instanceof IntLiteral),
-                        "2nd argument of function round/floor/ceil/truncate must be literal");
+                        "2nd argument of function round/floor/ceil must be literal");
```

Comment:
why not support all round family function? if truncate support slot as 2nd arg, other round family function should support this feature too

Reply:
> why not support all round family function? if truncate support slot as 2nd arg, other round family function should support this feature too

Need extra work and many tests on be, maybe one extra week.

This PR completes a lot of basic work, support for other functions will be much easier, so I list them as TODO task.

https://github.com/apache/doris/issues/32755